src/main/java/org/snomed/snowstorm/core/data/services/identifier/IdentifierCacheManager.java

```diff
@@ -194,6 +216,7 @@ public void stopBackgroundTask() {
 		stayAlive = false;
 		cacheDaemon.interrupt();
 		cacheDaemon = null;
+		identifierCaches.clear();
```

Comment:
I added this line because the tests in IdentifierCacheManagerTest were interfering with each other.

Reply:
I didn't realised this was within `stopBackgroundTask`, that's fine 👍